.gitignore

```diff
@@ -131,3 +131,5 @@ artifacts
 
 yarn.lock
 !/yarn.lock
+
+admin.schema.json
```

Comment:
> since I'm not sure if we should be publishing this schema here

👍🏻

Since we're targeting the unstable branch we don't want this to look like developers can dip into it and use it on their own